### PR TITLE
github actions files & readMe

### PR DESCRIPTION
**Pull request by SharonnElfride**

### CODE REVIEW

File: .github/workflows/delete-branch-after-merging.yml.OLD [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: GitHub Actions - Delete head branch after a pull request is closed
```
Comment (gaetanlhf) [inline .github/workflows/delete-branch-after-merging.yml.OLD:1]: This file will not be used...
Keep it anyway?

File: .github/workflows/merge-main-to-develop-after-push.yml.OLD [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: GitHub Actions - Merge modifications made on main to develop
```
Comment (gaetanlhf) [inline .github/workflows/merge-main-to-develop-after-push.yml.OLD:1]: This file will not be used...
Keep it anyway?

**Review by gaetanlhf — APPROVED** [review]

File: .github/workflows/delete-branch-after-merging.yml.OLD (hunk 1 above)
Comment (SharonnElfride) [reply .github/workflows/delete-branch-after-merging.yml.OLD:1]: Yeah, I want to work on it in the following days. If it still is not necessary at all, I will delete it.

File: .github/workflows/merge-main-to-develop-after-push.yml.OLD (hunk 2 above)
Comment (SharonnElfride) [reply .github/workflows/merge-main-to-develop-after-push.yml.OLD:1]: Yeah, I want to work on it in the following days. If it still is not necessary at all, I will delete it

File: .github/workflows/delete-branch-after-merging.yml.OLD (hunk 1 above)
Comment (gaetanlhf) [reply .github/workflows/delete-branch-after-merging.yml.OLD:1]: Okay 👍

File: .github/workflows/merge-main-to-develop-after-push.yml.OLD (hunk 2 above)
Comment (gaetanlhf) [reply .github/workflows/merge-main-to-develop-after-push.yml.OLD:1]: Okay 👍

File: .github/workflows/sonalCloud-build.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: SonarCloud Build 🚀
```
Comment (gaetanlhf) [inline .github/workflows/sonalCloud-build.yml:1]: .github/workflows/sonalCloud-build.yml
-> .github/workflows/SonarCloud-build.yml

**Review by gaetanlhf — CHANGES_REQUESTED** [review]

File: .github/workflows/sonalCloud-build.yml (hunk 3 above)
Comment (SharonnElfride) [reply .github/workflows/sonalCloud-build.yml]: Is .github/workflows/sonarCloud-build.yml alright ?

**Review by gaetanlhf — APPROVED** [review]

**Pull request merged**